<?xml version="1.0" encoding="UTF-8"?>
<testsuite xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://maven.apache.org/surefire/maven-surefire-plugin/xsd/surefire-test-report.xsd" name="tasksFuncional.taskTest" time="10.171" tests="4" errors="4" skipped="0" failures="0">
  <properties>
    <property name="sun.desktop" value="windows"/>
    <property name="awt.toolkit" value="sun.awt.windows.WToolkit"/>
    <property name="java.specification.version" value="11"/>
    <property name="sun.cpu.isalist" value="amd64"/>
    <property name="sun.jnu.encoding" value="Cp1252"/>
    <property name="java.class.path" value="C:\Maven\boot\plexus-classworlds-2.6.0.jar"/>
    <property name="java.vm.vendor" value="Amazon.com Inc."/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="user.variant" value=""/>
    <property name="java.vendor.url" value="https://aws.amazon.com/corretto/"/>
    <property name="user.timezone" value="America/Sao_Paulo"/>
    <property name="maven.conf" value="C:\Maven/conf"/>
    <property name="os.name" value="Windows 10"/>
    <property name="java.vm.specification.version" value="11"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="user.country" value="BR"/>
    <property name="sun.boot.library.path" value="C:\Program Files\Amazon Corretto\jdk11.0.12_7\bin"/>
    <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher clean test"/>
    <property name="jdk.debug" value="release"/>
    <property name="maven.home" value="C:\Maven"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="user.home" value="C:\Users\welli"/>
    <property name="user.language" value="pt"/>
    <property name="sun.stderr.encoding" value="cp850"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="java.version.date" value="2021-07-20"/>
    <property name="java.home" value="C:\Program Files\Amazon Corretto\jdk11.0.12_7"/>
    <property name="file.separator" value="\"/>
    <property name="java.vm.compressedOopsMode" value="Zero based"/>
    <property name="line.separator" value="&#10;"/>
    <property name="sun.stdout.encoding" value="cp850"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="java.awt.graphicsenv" value="sun.awt.Win32GraphicsEnvironment"/>
    <property name="user.script" value=""/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="java.runtime.version" value="11.0.12+7-LTS"/>
    <property name="user.name" value="welli"/>
    <property name="path.separator" value=";"/>
    <property name="os.version" value="10.0"/>
    <property name="java.runtime.name" value="OpenJDK Runtime Environment"/>
    <property name="file.encoding" value="Cp1252"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="java.vm.name" value="OpenJDK 64-Bit Server VM"/>
    <property name="java.vendor.version" value="Corretto-11.0.12.7.1"/>
    <property name="java.vendor.url.bug" value="https://github.com/corretto/corretto-11/issues/"/>
    <property name="java.io.tmpdir" value="C:\Users\welli\AppData\Local\Temp\"/>
    <property name="java.version" value="11.0.12"/>
    <property name="user.dir" value="C:\Devops\funcionalTest"/>
    <property name="os.arch" value="amd64"/>
    <property name="maven.multiModuleProjectDirectory" value="C:\Devops\funcionalTest"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="java.awt.printerjob" value="sun.awt.windows.WPrinterJob"/>
    <property name="sun.os.patch.level" value=""/>
    <property name="java.library.path" value="C:\Program Files\Amazon Corretto\jdk11.0.12_7\bin;C:\WINDOWS\Sun\Java\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\DPOS8\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Docker\Docker\resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Program Files\Git\cmd;C:\Maven\bin;C:\Program Files\Amazon Corretto\jdk11.0.12_7\bin;C:\ProgramData\chocolatey\bin;C:\Users\welli\AppData\Local\Android\Sdk\platform-tools;C:\Program Files\nodejs\;C:\Program Files\dotnet\;C:\Users\welli\Downloads\seleniumRenner-master\seleniumRenner-master\target\ChromeDriver;C:\Program Files\Amazon Corretto\jdk11.0.12_7\bin;C:\Program Files\Amazon Corretto\jdk11.0.12_7;C:\DPOS8\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Docker\Docker\resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Program Files\Git\cmd;C:\Maven\bin;C:\Program Files\Amazon Corretto\jdk11.0.12_7\bin;C:\ProgramData\chocolatey\bin;C:\Users\welli\AppData\Local\Android\Sdk\platform-tools;C:\Program Files\nodejs\;C:\Program Files\dotnet\;C:\Impulse\Projetos\DEVImpulse\Automacao\Driver\chromedriver.exe;C:\Users\welli\AppData\Local\Microsoft\WindowsApps;;C:\Users\welli\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\welli\AppData\Roaming\npm;C:\Impulse\Projetos\DEVImpulse\Automacao\Driver;."/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.vendor" value="Amazon.com Inc."/>
    <property name="java.vm.version" value="11.0.12+7-LTS"/>
    <property name="classworlds.conf" value="C:\Maven\bin\m2.conf"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="library.jansi.path" value="C:\Maven\lib\jansi-native"/>
    <property name="java.class.version" value="55.0"/>
  </properties>
  <testcase name="deveSalvarTarefaComSucesso" classname="tasksFuncional.taskTest" time="8.277">
    <error message="unknown error: net::ERR_CONNECTION_REFUSED&#10;  (Session info: chrome=100.0.4896.75)&#10;Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;&#10;System info: host: &apos;WELLINGTON&apos;, ip: &apos;192.168.100.82&apos;, os.name: &apos;Windows 10&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;11.0.12&apos;&#10;Driver info: org.openqa.selenium.chrome.ChromeDriver&#10;Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 100.0.4896.75, chrome: {chromedriverVersion: 100.0.4896.60 (6a5d10861ce8..., userDataDir: C:\Users\welli\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:60843}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}&#10;Session ID: 338e4c2a906982c83cb3c09d5d1f01d6" type="org.openqa.selenium.WebDriverException">org.openqa.selenium.WebDriverException: 
unknown error: net::ERR_CONNECTION_REFUSED
  (Session info: chrome=100.0.4896.75)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'WELLINGTON', ip: '192.168.100.82', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.12'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 100.0.4896.75, chrome: {chromedriverVersion: 100.0.4896.60 (6a5d10861ce8..., userDataDir: C:\Users\welli\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:60843}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 338e4c2a906982c83cb3c09d5d1f01d6
	at tasksFuncional.taskTest.acessarAplicacao(taskTest.java:25)
	at tasksFuncional.taskTest.deveSalvarTarefaComSucesso(taskTest.java:56)
</error>
    <system-err><![CDATA[Starting ChromeDriver 100.0.4896.60 (6a5d10861ce8de5fce22564658033b43cb7de047-refs/branch-heads/4896@{#875}) on port 25052Only local connections are allowed.Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.ChromeDriver was started successfully.abr 06, 2022 11:25:34 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFORMAÇÕES: Detected dialect: W3C
]]></system-err>
  </testcase>
  <testcase name="deveSalvarTarefaComDataPassada" classname="tasksFuncional.taskTest" time="10.17">
    <error message="unknown error: net::ERR_CONNECTION_REFUSED&#10;  (Session info: chrome=100.0.4896.75)&#10;Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;&#10;System info: host: &apos;WELLINGTON&apos;, ip: &apos;192.168.100.82&apos;, os.name: &apos;Windows 10&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;11.0.12&apos;&#10;Driver info: org.openqa.selenium.chrome.ChromeDriver&#10;Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 100.0.4896.75, chrome: {chromedriverVersion: 100.0.4896.60 (6a5d10861ce8..., userDataDir: C:\Users\welli\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:60858}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}&#10;Session ID: a13b362a6375294afdc285e3901ada71" type="org.openqa.selenium.WebDriverException">org.openqa.selenium.WebDriverException: 
unknown error: net::ERR_CONNECTION_REFUSED
  (Session info: chrome=100.0.4896.75)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'WELLINGTON', ip: '192.168.100.82', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.12'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 100.0.4896.75, chrome: {chromedriverVersion: 100.0.4896.60 (6a5d10861ce8..., userDataDir: C:\Users\welli\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:60858}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: a13b362a6375294afdc285e3901ada71
	at tasksFuncional.taskTest.acessarAplicacao(taskTest.java:25)
	at tasksFuncional.taskTest.deveSalvarTarefaComDataPassada(taskTest.java:107)
</error>
    <system-err><![CDATA[Starting ChromeDriver 100.0.4896.60 (6a5d10861ce8de5fce22564658033b43cb7de047-refs/branch-heads/4896@{#875}) on port 42189Only local connections are allowed.Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.ChromeDriver was started successfully.abr 06, 2022 11:25:36 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFORMAÇÕES: Detected dialect: W3C
]]></system-err>
  </testcase>
  <testcase name="deveSalvarTarefaSemDescricao" classname="tasksFuncional.taskTest" time="8.143">
    <error message="unknown error: net::ERR_CONNECTION_REFUSED&#10;  (Session info: chrome=100.0.4896.75)&#10;Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;&#10;System info: host: &apos;WELLINGTON&apos;, ip: &apos;192.168.100.82&apos;, os.name: &apos;Windows 10&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;11.0.12&apos;&#10;Driver info: org.openqa.selenium.chrome.ChromeDriver&#10;Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 100.0.4896.75, chrome: {chromedriverVersion: 100.0.4896.60 (6a5d10861ce8..., userDataDir: C:\Users\welli\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:60836}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}&#10;Session ID: 2f21b212d96a42abc38b237061361b3d" type="org.openqa.selenium.WebDriverException">org.openqa.selenium.WebDriverException: 
unknown error: net::ERR_CONNECTION_REFUSED
  (Session info: chrome=100.0.4896.75)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'WELLINGTON', ip: '192.168.100.82', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.12'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 100.0.4896.75, chrome: {chromedriverVersion: 100.0.4896.60 (6a5d10861ce8..., userDataDir: C:\Users\welli\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:60836}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 2f21b212d96a42abc38b237061361b3d
	at tasksFuncional.taskTest.acessarAplicacao(taskTest.java:25)
	at tasksFuncional.taskTest.deveSalvarTarefaSemDescricao(taskTest.java:73)
</error>
    <system-err><![CDATA[Starting ChromeDriver 100.0.4896.60 (6a5d10861ce8de5fce22564658033b43cb7de047-refs/branch-heads/4896@{#875}) on port 33415Only local connections are allowed.Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.ChromeDriver was started successfully.abr 06, 2022 11:25:33 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFORMAÇÕES: Detected dialect: W3C
]]></system-err>
  </testcase>
  <testcase name="deveSalvarTarefaSemData" classname="tasksFuncional.taskTest" time="9.629">
    <error message="unknown error: net::ERR_CONNECTION_REFUSED&#10;  (Session info: chrome=100.0.4896.75)&#10;Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;&#10;System info: host: &apos;WELLINGTON&apos;, ip: &apos;192.168.100.82&apos;, os.name: &apos;Windows 10&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;11.0.12&apos;&#10;Driver info: org.openqa.selenium.chrome.ChromeDriver&#10;Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 100.0.4896.75, chrome: {chromedriverVersion: 100.0.4896.60 (6a5d10861ce8..., userDataDir: C:\Users\welli\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:60840}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}&#10;Session ID: 342c7e2028cc4456217454e4563c248b" type="org.openqa.selenium.WebDriverException">org.openqa.selenium.WebDriverException: 
unknown error: net::ERR_CONNECTION_REFUSED
  (Session info: chrome=100.0.4896.75)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'WELLINGTON', ip: '192.168.100.82', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.12'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 100.0.4896.75, chrome: {chromedriverVersion: 100.0.4896.60 (6a5d10861ce8..., userDataDir: C:\Users\welli\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:60840}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 342c7e2028cc4456217454e4563c248b
	at tasksFuncional.taskTest.acessarAplicacao(taskTest.java:25)
	at tasksFuncional.taskTest.deveSalvarTarefaSemData(taskTest.java:90)
</error>
    <system-err><![CDATA[Starting ChromeDriver 100.0.4896.60 (6a5d10861ce8de5fce22564658033b43cb7de047-refs/branch-heads/4896@{#875}) on port 7171Only local connections are allowed.Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.ChromeDriver was started successfully.abr 06, 2022 11:25:35 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFORMAÇÕES: Detected dialect: W3C
]]></system-err>
  </testcase>
</testsuite>